website/static/worker.js

```diff
@@ -40,8 +40,11 @@ self.require = function require(path) {
   return self[path];
 };
 
+var prettier;
 importScripts("lib/index.js");
-var prettier = index; // eslint-disable-line
+if (typeof prettier === "undefined") {
+  prettier = index; // eslint-disable-line
```

Comment:
It seems it's called `repo`,  don't know why.

Reply:
Ah, must have been the rollup-plugin-commonjs upgrade.

Reply:
Yep, that worked! :taco: